javascript/javascript2/week1/spiritAnimal.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <main>
+        <h1>Spirit Animal Name Generator</h1>
+
+        <div class="buttons">
+            <h2>
+                How would you like your Spirit Animal name to be generated?
+            </h2>
+            <p>
+                <button id="withbutton">When I push the button</button>
+                <button id="withhover">When I Hover the input</button>
+                <button id="withtext">When I write text</button>
+            </p>
+        </div>
+
+        <div class="result">
+            <label for="username">Username</label>
+            <input type="text" id="username">
+            <button id="myButton">Generate Spirit Animal</button>
```

Comment:
This is not a "myButton", it might be a "submitButton" or "generateButton".

Reply:
Thanks I will fix that.